unityfiles/Assets/Scripts/Game/Spawning/SimpleSpawner.cs

```diff
@@ -28,16 +28,14 @@ public class SimpleSpawner : MonoBehaviour {
 
         SpriteRenderer spriteRender = ebp.AddComponent<SpriteRenderer> ();
         spriteRender.sprite = Sprite.Create (this.enemyTexture, new Rect (0, 0, this.enemyTexture.width, this.enemyTexture.height), Vector2.zero);
-        ebp.transform.localScale = new Vector3 (0.25f, 0.25f, 1);
+        ebp.transform.localScale = new Vector3 (0.025f, 0.025f, 1);
 
         BoxCollider2D bc = ebp.AddComponent<BoxCollider2D> ();
 
         Enemy enemyScript = ebp.AddComponent<Enemy> () as Enemy;
-        // print(this.enemyTypeToSpawn);
-        // print(ebp.GetComponent<Enemy>().EnemyType);
+
         enemyScript.EnemyType = this.enemyTypeToSpawn;
-        // print(this.enemyTypeToSpawn);
-        // print(this.enemyBlueprint.GetComponent<Enemy>().EnemyType);
+      
 
         IEnemyBehavior EBH = ebp.AddComponent<EBTestMoveDown> ();
```

Comment:
What is ebp, and maby change "EBH" to something more readable

Reply:
enemy blue print den blir litt lang til og skrive ut siden den er en temporary variabel

Reply:
`bc` for box collider gir seg selv, men `ebp` for enemy blue print, not so much. Jeg trodde det stod for enemy behavior pattern. Prøv å unngå slike ting da det lett kan bli forvirrende når vi bare ser fort over koden.

Reply:
jeg mener det blir veldig mye tekst men ok